src/Psr7/RequestTransformer.php

```diff
@@ -15,12 +15,6 @@ class RequestTransformer
      */
     public function toPsr7(Request $request)
     {
-        // Prevent DiactorosFactory::createRequest from locking
-        // the content out from subsequent Request client code. This hack
-        // is needed until https://github.com/symfony/symfony/pull/19549
-        // finds its way into a future Symfony release.
-        $request->getContent();
```

Comment:
Yes, otherwise we'd have to muck around with the composer.json file so that for instance 2.7.20 is accepted but 2.8.2 or 3.1.3 are not. I'd rather keep this line and the sane dependency requirements (i.e. ^2.7 and above works).